2018-pydays-vienna/MLonCode.md

```diff
@@ -8,6 +8,10 @@
 
 **Author:** Vadim
 
+**Slides:** https://warenlg.github.io/pydays-vienna-2018/
```

Comment:
Correct link to Vadim's slides http://vmarkovtsev.github.io/pydays-2018-vienna/#